docs-src/content/using-cubic-sdk/streaming.md

```diff
@@ -457,14 +457,19 @@ Simply pass the bytes from the microphone the same as is done from the file in t
 
 {{< /tab >}}
 
+{{< tab "C++" "c++" >}}
+
+For a complete C++ example, see [the examples-cpp github repository](https://github.com/cobaltspeech/examples-cpp).
+{{< /tab >}}
```

Comment:
Is there a section on C# that we should link up?

Reply:
Yes, but I'll make more documentation changes in a separate PR because Shahruk has other changes to that section in a different branch and I didn't want to complicate that merge